.gitignore

```diff
@@ -1,7 +1,7 @@
 *.swp
 *.iml
 .idea
-*.iml
+vendor/
```

Comment:
We will not remove vendor right now, you should add your new dependencies to vendor. Take a look at make mod target.